File: app/src/test/java/com/mss/instamat/repositories/files/FilesHelperTest.java
package com.mss.instamat.repositories.files;

import com.mss.instamat.domain.models.ImageInfo;

import org.junit.Test;

import static org.junit.Assert.*;

public class FilesHelperTest {

    @Test
    public void getFilePath() {
        ImageInfo imageInfo = new ImageInfo();
        imageInfo.setId(1);
        FilesHelper filesHelper = new FilesHelper();

        assertEquals("path\\1.jpg", filesHelper.getFilePath("path", imageInfo));
    }
}

File: app/src/test/java/com/mss/imagesearcher/model/repositories/files/FilesHelperTest.kt
package com.mss.imagesearcher.model.repositories.files

import com.mss.imagesearcher.model.entity.ImageInfo
import org.junit.Assert.assertEquals
import org.junit.Test
import java.io.File

class FilesHelperTest {

    @Test
    fun getFilePath() {
        val imageInfo = ImageInfo()
        imageInfo.id = 1
        val filesHelper = FilesHelper()

        assertEquals("path" + File.separator + "1.jpg", filesHelper.getFilePath("path", imageInfo))
    }
}